clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -158,6 +158,10 @@ static SYCLBIN::BundleState SYCLBINState = SYCLBIN::BundleState::Input;
 
 static SmallString<128> OffloadImageDumpDir;
 
+static std::string SYCLCompileOptionsFromImage;
```

Comment:
Any specific reason why this is a `std::string` and not a `StringRef`?

Reply:
Data-owning containers are more reliable against use-after-free bugs.